### ISSUE TITLE
Basic commands "tcpip" and "devices" (for example)

### ISSUE THREAD

**Issue by Murffyp** [KhunHtetzNaing/ADB-OTG#69]
Hi tcpip command seems inaccessible through your app..  any way to get around that it or fix it.... It's one of the main commands I need as every time I reboot my devices I need to run it so other things work (e.g. tasker).

I'd appreciate any help/fixed you can pass along.

Thanks!!!

**Comment by Murffyp** [comment]
???

**Comment by benstaf** [comment]
Yes that would be great to have a feature similar to Bugjaeger, which is able to perform the equivalent to adb tcpip 5555 in one click, even with a unrooted android phone, which is quite unique feature (Termux can't, and all adb shell apps can't without root)

![Image](https://github.com/user-attachments/assets/838b8c41-2e73-4a6c-9788-9c674c04e844)
![Image](https://github.com/user-attachments/assets/960484c5-ce55-417f-b0d8-b9969d90d1eb)
